src/lib/storage/fixed_size_attribute_vector.cpp

```diff
@@ -0,0 +1,49 @@
+// the linter wants this to be above everything else
+#include "fixed_size_attribute_vector.hpp"
+#include <boost/numeric/conversion/cast.hpp>
+#include <types.hpp>
+#include <vector>
+
+namespace opossum {
+
+template <typename T>
+FixedSizeAttributeVector<T>::FixedSizeAttributeVector(const size_t size) {
+  _values = std::vector<T>(size);
+}
+
+// returns the value id at a given position
+template <typename T>
+ValueID FixedSizeAttributeVector<T>::get(const size_t i) const {
+  return ValueID{_values.at(i)};
+}
+
+// sets the value id at a given position
+template <typename T>
+void FixedSizeAttributeVector<T>::set(const size_t i, const ValueID value_id) {
+  _values[i] = value_id;
```

Comment:
Check if the position exists or use .at() instead of []

Reply:
I wouldn't necessarily agree, see above